src/Main.gren

```diff
@@ -40,6 +41,7 @@ type alias Model =
     , stderr : Stream.Writable Bytes
     , server : Maybe HttpServer.Server
     , db : Db.Connection
+    , secureContext : Maybe Crypto.SecureContext
```

Comment:
Is it possible to start the program with `Init.awaitTask (Task.map3 Registry.Db.migrate HttpServer.createServer Crypto.getSecureContext |> Task.onError ...)` in order to avoid storing `Maybe`'s in the model? 🤔